instrumentation/spring/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -276,7 +276,13 @@
     {
       "name": "otel.instrumentation.common.db-statement-sanitizer.enabled",
       "type": "java.lang.Boolean",
-      "description": "Enables the DB statement sanitization for R2DBC.",
+      "description": "Enables the DB statement sanitization for Database (JDBC and R2DBC currently).",
```

Comment:
I'd just use something generic like `Enables the DB statement sanitization.` to avoid needing updating this when new instrumentations are added.